frontend/src/components/Question.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import { ChevronLeft, ChevronRight, ExpandMore } from "@mui/icons-material";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Divider,
+  Grid,
+  IconButton,
+  Paper,
+  Stack,
+  Tooltip,
+  Typography,
+} from "@mui/material";
+
+import { Center } from "../components/Center";
+
+export const Question = () => {
+  const [expanded, setExpanded] = useState<boolean>(true);
+
+  const handleChange = () => {
+    setExpanded(!expanded);
+  };
+
+  return expanded ? (
+    <Grid item xs={4}>
+      <Paper elevation={1} sx={{ p: 3 }}>
+        <Stack direction="column" spacing={3}>
+          <Center>
```

Comment:
Is there a way to position this such that the typography is centered, but the icon button is flex-end?